### DISCUSSION TITLE
updated

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#4752] [General]
Guys, can someone please give advice on how the Wiki is being updated?

**Comment by tommoor** [comment]
I'm not sure what you mean

**Reply by MaksGill** [reply]
Outline is constantly updated, which is good, it has different versions. Where can I download these new versions?

**Reply by tommoor** [reply]
Releases are distributed through Docker: https://hub.docker.com/repository/docker/outlinewiki/outline

**Reply by MaksGill** [reply]
Thanks a lot for the answer.

**Comment by logger55** [comment]
hello! outline was installed from source https://docs.getoutline.com/s/hosting/doc/from-source-BlBxrNzMIP
how can i update it ? there is no update instructions on the wiki
do i need to update every new release sequentially ? outline 0.61 -  26 new releases have been released since installation.
do i need to download the release files and replace them with them ?
what update commands should be run after replacing files?
may be 
yarn install --frozen-lockfile && yarn build
yarn sequelize db:migrate --env=production-ssl-disabled ?
